modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/Activities/index.scss

```diff
@@ -1,135 +1,153 @@
-.border-link-active::after {
-	background-color: var(--primary);
-	content: '';
-	display: block;
-	height: 1px;
-	margin-top: 6px;
-	position: absolute;
-	width: 25px;
-}
-
-.blue-line-activites:after {
-	background-color: #4c84ff;
-	bottom: -1px;
-	content: '';
-	display: block;
-	left: -1px;
-	position: absolute;
-	top: 0;
-	width: 5px;
-}
-
-.box-activites {
-	height: 53px;
-}
+.table-list-container {
+	.border-link-active::after {
+		background-color: var(--primary);
+		content: '';
+		display: block;
+		height: 1px;
+		margin-top: 6px;
+		position: absolute;
+		width: 25px;
+	}
 
-.box-table {
-	tbody {
-		display: flex;
-		flex-direction: column;
-		height: 230px;
-		overflow: auto;
-		width: 100%;
+	.blue-line-activites:after {
+		background-color: #4c84ff;
+		bottom: -1px;
+		content: '';
+		display: block;
+		left: -1px;
+		position: absolute;
+		top: 0;
+		width: 5px;
 	}
-	thead {
-		display: table;
-		width: 100%;
+
+	.box-activites {
+		height: 53px;
 	}
-	tbody tr {
-		display: table;
-		width: 100%;
+
+	.box-table {
+		tbody {
+			display: flex;
+			flex-direction: column;
+			height: 230px;
+			overflow: auto;
+			width: 100%;
+		}
+
+		thead {
+			display: flex;
+		}
+
+		thead tr {
+			display: flex;
+			justify-content: space-between;
+			padding-right: 90px;
+			width: 100%;
+		}
+		tbody tr {
+			display: flex;
+			justify-content: space-between;
+			width: 100%;
+		}
 	}
-}
 
-.box-table-mobile {
-	tbody {
-		display: flex;
-		flex-direction: column;
-		height: 100%;
-		width: 100%;
+	.box-table-mobile {
+		thead {
+			display: table;
+			width: 100%;
+		}
+		tbody tr {
+			justify-content: space-between;
+			width: 100%;
+		}
 	}
-	thead {
-		display: table;
-		width: 100%;
+
+	.bg-message {
+		background-color: #fff;
+		border-bottom-left-radius: 0.25rem;
+		border-bottom-right-radius: 0.25rem;
 	}
```

Comment:
Try to use bootstrap Borders and dialect colors here.

Reply:
Done. THx!